hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestSchemaEvolution.java

```diff
@@ -232,11 +242,12 @@ private void changeTableSchema(TableOptions tableOptions, boolean shouldCompactB
       }
 
       Schema intType = SchemaBuilder.unionOf().nullType().and().intType().endUnion();
+      Schema longType = SchemaBuilder.unionOf().nullType().and().longType().endUnion();
       Schema doubleType = SchemaBuilder.unionOf().nullType().and().doubleType().endUnion();
       Schema stringType = SchemaBuilder.unionOf().nullType().and().stringType().endUnion();
       Schema structType = SchemaBuilder.builder().record("new_row_col").fields()
-              .name("f0").type("long").noDefault()
-              .name("f1").type("string").noDefault().endRecord();
+              .name("f0").type(longType).noDefault()
+              .name("f1").type(stringType).noDefault().endRecord();
```

Comment:
is this a mistake before?

Reply:
I believe so. The nullability of column fields in `new_row_col` here (`changeTableSchema`) is not consistent with that in `writeTableWithSchema2`.